internal/applicationsnapshot/report.go

```diff
@@ -30,13 +31,50 @@ type Component struct {
 	Success    bool            `json:"success"`
 }
 
-type report struct {
+type fullReport struct {
```

Comment:
I think the struct can still be called report...
```suggestion
type report struct {
```